certified-connectors/Seismic Workspace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2366 @@
+        "description": "Get the comments on an item along with any replies and annotations.",
+        "operationId": "GetWorkspaceItemComments",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "workspaceContentId",
+            "description": "Workspace Content Id",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Workspace Content Id"
+          },
+          {
+            "in": "path",
+            "name": "spaceId",
+            "description": "space Id",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Space Id"
+          },
+          {
+            "in": "query",
+            "name": "versionId",
+            "description": "Workspace Content Version Id",
+            "type": "string",
+            "x-ms-summary": "Version Id"
+          },
+          {
+            "in": "query",
+            "name": "offset",
+            "description": "offset",
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "Offset"
+          },
+          {
+            "in": "query",
+            "name": "limit",
+            "description": "limit",
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "Limit"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/Seismic.Paging.WorkspaceComment.WorkspaceComment"
+            }
+          }
+        }
+      }
+    },
+    "/workspace/spaces/{spaceId}/items/{workspaceContentId}/versions/{workspaceVersionId}/comments": {
+      "post": {
+        "tags": [
+          "Workspace Comments"
+        ],
+        "summary": "Add a comment on an item",
+        "operationId": "AddWorkspaceItemComments",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "spaceId",
+            "description": "Space Id",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Space Id"
+          },
+          {
+            "in": "path",
+            "name": "workspaceContentId",
+            "description": "Workspace Content Id",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Workspace Content Id"
+          },
+          {
+            "in": "path",
+            "name": "workspaceVersionId",
+            "description": "Workspace Content Version Id",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Workspace Version Id"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "",
```

Comment:
@rkushvaha can you please define the description for the parameter. please fix empty descriptions in the swagger file.

Reply:
@Amjed-Ayoub updated